src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -2886,6 +2886,48 @@ GUI_TEST_CLASS_DEFINITION(test_7623) {
     GTUtilsLog::checkContainsError(os, logTracer, "All input reads contain gaps or Ns only, abort");
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7629) {
+    //1. Open sars.gb
+    GTFileDialog::openFile(os, dataDir + "/samples/Genbank/sars.gb");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+    //2. Copy whole sequence
+    GTUtilsDialog::waitForDialog(os, new SelectSequenceRegionDialogFiller(os, 1, 29751));
+    GTUtilsDialog::waitForDialog(os, new PopupChooser(os, {"Select", "Sequence region"}));
+    GTMenu::showContextMenu(os, GTWidget::findWidget(os, "ADV_single_sequence_widget_0"));
+
+    GTUtilsDialog::waitForDialog(os, new PopupChooserByText(os, QStringList() << "Copy/Paste" << "Copy selected sequence"));
+    GTMenu::showContextMenu(os, GTWidget::findWidget(os, "ADV_single_sequence_widget_0"));
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+    //3. Paste it to project filter
+    //Expected: no crash, here is error in log
+    GTLogTracer logTracer;
+    auto nameFilterEdit = GTWidget::findLineEdit(os, "nameFilterEdit");
+    GTLineEdit::setText(os, nameFilterEdit, GTClipboard::text(os), true, true);
+    //necessary sleep to activate search task(s) after pasting sequence
+    GTGlobals::sleep();
```

Comment:
Use checkTaskIsPresent if you need to wait for a task.

Reply:
Why this comment was resolved with no changes?

Reply:
It is not possible to replace first sleep() occurrence with checkTaskIsPresent(), because with too long pattern, the task disappears from the tree too quickly for the method to catch it.
[build with both sleep() replaced](http://ugene-teamcity.unipro.ru:8111/ugene-teamcity/buildConfiguration/GuiLinux/249000)

Reply:
In this case replace with some other targeted waiter, like waitTasksFinished